### ISSUE TITLE
The focus jumps to the last group in the group menu after selecting/removing an item with keyboard(return & space key)

### ISSUE THREAD

**Issue by Changsuwan** [SAP/spartacus#11560]

**Comment by Uli-Tiger** [comment]
same root cause as #11563 and will be fixed together with it

**Comment by Uli-Tiger** [comment]
test on team server is missing - can be done tomorrow.
